deploy/kicbase/Dockerfile

```diff
@@ -162,9 +170,9 @@ RUN export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/') && \
     curl -LO https://downloadcontent.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_20.04/Release.key && \
     apt-key add - < Release.key && \
     if [ "$ARCH" != "ppc64le" ]; then \
-    	clean-install containers-common catatonit conmon containernetworking-plugins cri-tools podman-plugins crun; \
+       clean-install containers-common conmon containernetworking-plugins cri-tools crun podman-plugins catatonit; \
```

Comment:
Don't really see how this change is related to cri-dockerd ?

Reply:
it's not, i was debugging other issues related to the openSUSE repo and remove then added back some packages, this can be reverted.